src/RepoName.sln.DotSettings

```diff
@@ -1,6 +1,6 @@
 ﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
 
-	<s:String x:Key="/Default/CodeInspection/CSharpLanguageProject/LanguageLevel/@EntryValue">Experimental</s:String>
+	<s:String x:Key="/Default/CodeInspection/CSharpLanguageProject/LanguageLevel/@EntryValue">CSharp72</s:String>
```

Comment:
should we leave that at experimental? or maybe set to latest?

Reply:
Do we know what R# actually considers valid values for that setting? Any way for you to investigate? I'm not sure `latest` is valid there.

Reply:
I haven't found official doco on this, it's basically searching through the R# settings menu. I'll try to look at this a bit closer.

Reply:
Why not `CSharp73`?

Reply:
@adamralph When I did a bit of investigation earlier, I don't think R# supports 7.3 yet.

Reply:
If there is a way to just say "use the latest version you know about" that would be better than locking to a specific version and needing to update this all the time, especially since we just use `latest` for LangVersion in VS.

The danger of course is that R# lags behind the latest language features, so we could run into a problem if we try to use something it doesn't know about yet.

Reply:
Seems like `Default` is the setting we want to use, which let's R# infer the language level from the project file:

> The Default value means that ReSharper automatically detects the language version based on the project settings 

https://www.jetbrains.com/help/resharper/Configuring_Project_Specific_Properties.html

I've tested this with the NServiceBus.Core sln.DotSettings file and this seems to correctly accept 7.1 language features.

Reply:
I wish I knew what "project settings" it was using to detect the language version. `Default` seems like the setting we'd want, as long as it doesn't incorrectly infer the right value!

Reply:
@bording haven't found much info about that, but at first glance it seems to behave as expected (at least with the .net core sdk csproj style).